examples/esp_idf/README.md

```diff
@@ -51,3 +51,38 @@ If you're using an Adafruit Feather board with a USB-C connector,
 you will need to set `CONFIG_ESP_CONSOLE_USB_SERIAL_JTAG` to `y` via
 `sdkconfig` or `idf.py menuconfig`. This is required in order to use
 the command-line shell that is integrated with the ESP-IDF examples.
+
+
+### Testing the samples using pytest
+
+The pytests require:
+- An attached ESP board
+- The project's api key
+- The name of the serial port to which the board is attached
+- WiFi credentials to be present in a YAML file with the following format:
```

Comment:
WiFi credentials should be passed using command line options now that #341 is merged.